react/Alerter/index.jsx

```diff
@@ -3,7 +3,7 @@ import classNames from 'classnames'
 
 import styles from './index.styl'
 
-const AUTOCLOSE_DELAY = 2000
+const DEFAULT_AUTOCLOSE_DELAY = 3000
```

Comment:
3 seconds seems pretty short to me. It may feel long during development session but when you got a real notification during navigation, I think it's too short. What think @kelukelu or @jsilvestre about that ?